pkg/bucketeer/model/reason.go

```diff
@@ -36,4 +36,6 @@ const (
 	ReasonErrorFeatureFlagIDNotSpecified ReasonType = "ERROR_FEATURE_FLAG_ID_NOT_SPECIFIED"
 	// An unexpected error occurred during evaluation.
 	ReasonErrorException ReasonType = "ERROR_EXCEPTION"
+	// The flag is missing in the cache; the default value was returned.
```

Comment:
The flag is missing in the cache or in backend should use FLAG_NOT_FOUND instead.
The CACHE_NOT_FOUND is used when after the SDK is initialized but there is no cache.
This can happen if the user call the get variation interface right after initialized the SDK without waiting the background task to update the cache.
This could happen when a pod is initializing and when it's ready, the cache might not b there yet and the pod receives requests.